src/sentry/tasks/post_process.py

```diff
@@ -524,7 +525,11 @@ def post_process_group(
                 )
                 return
             with metrics.timer("tasks.post_process.delete_event_cache"):
-                event_processing_store.delete_by_key(cache_key)
```

Comment:
do you also need to update line 520 to point at the transacations store?

Reply:
:+1: to what Lyn said. You could have a similar condition to what you had in [src/sentry/reprocessing2.py](https://github.com/getsentry/sentry/pull/79200/files#diff-96a11762040dd39019362a57f6c59d84f7568f1a11083c642509018a04158141) here, and then update all `event_processing_store` references to the generalized one.